runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/WorkUnitClient.java

```diff
@@ -49,4 +51,17 @@ interface WorkUnitClient {
    * @return a {@link WorkItemServiceState} (e.g. a new stop position)
    */
   WorkItemServiceState reportWorkItemStatus(WorkItemStatus workItemStatus) throws IOException;
+  /**
```

Comment:
blank line above to separate from previous method (I'm suprised spotlessApply doesn't care but might as well match the rest of the file).